tests/functional-tests/src/test/resources/GebConfig.groovy

```diff
@@ -37,10 +37,10 @@ if (!USERNAME || !AUTOMATE_KEY)
 
 // Wait Settings
 waiting {
-	timeout = 20
-	retryInterval = 1
+	timeout = 30
+	retryInterval = 3
 }
-atCheckWaiting = [20, 1]
+atCheckWaiting = [30, 3]
```

Comment:
Any way to have atCheckWaiting leverage the waiting.timeout/retryInterval values as variables instead? Granted they're right next to each other linewise, but it would be good to reference by variable if possible.

Also, this file is missing the (pedantic) License comment header.

Reply:
<waiting.timeout/retryInterval values as variables > : Typically , you will not touch these at all. If you need to touch them it is once. So I do no see the need to parameterize parameters that I set in the config file.